### ISSUE TITLE
problem with alias for ddgr

### ISSUE THREAD

**Issue by kurokirasama** [nushell/nushell#5267]
### Describe the bug

I have this alias:
```
alias gg = ddgr -n 5
```
But when I try it I get:
```
Error: nu::shell::external_command (link)

  × External command
   ╭─[entry #6:1:1]
 1 │ gg end of line regex
   · ─┬
   ·  ╰── can't run executable
   ╰────
  help: No such file or directory (os error 2)
```
I tried with parenthesis `alias gg = (ddgr -n 5)` but I got:

```
Error: nu::parser::parse_mismatch (link)

  × Parse mismatch during operation.
   ╭─[entry #3:1:1]
 1 │ gg nushell
   ·    ───┬───
   ·       ╰── expected operator
   ╰────
```

### How to reproduce

# Install ddgr (choose an install dir)
```
git clone https://github.com/jarun/ddgr.git
cd ddgr
sudo make install
sudo ddgr -u
```
# Create alias and add it to config.nu
```
alias gg = ddgr -n 5
```
# Try it:
```
gg nushel
```

### Expected behavior

it should give the output of `ddgr` command

### Configuration

| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.61.1                                   |
| branch             |                                          |
| tag                |                                          |
| short_commit       | f2d47f97                                 |
| commit_hash        | f2d47f97da15e52248dd285c8ba0e1bd37dd4de3 |
| commit_date        | 2022-04-19 19:11:58 +00:00               |
| build_os           | linux-x86_64                             |
| rust_version       | rustc 1.60.0 (7737e0b5c 2022-04-04)      |
| rust_channel       | stable-x86_64-unknown-linux-gnu          |
| cargo_version      | cargo 1.60.0 (d1fd9fe 2022-03-01)        |
| pkg_version        | 0.61.1                                   |
| build_time         | 2022-04-19 23:58:41 -04:00               |
| build_rust_channel | debug                                    |
| features           | dataframe, default, trash, which, zip    |
| installed_plugins  |                                          |

**Comment by jntrnr** [comment]
@kurokirasama for your example, can you share what line caused this?

```
Error: nu::shell::external_command (link)

  × External command
   ╭─[entry #6:1:1]
 1 │ gg end of line regex
   · ─┬
   ·  ╰── can't run executable
   ╰────
  help: No such file or directory (os error 2)
```

**Comment by kurokirasama** [comment]
what do you mean? what line in config.nu?

**Comment by fdncred** [comment]
do you get the same error if you define the alias like this `alias gg = (ddgr -n 5)`?

**Comment by kurokirasama** [comment]
yes

**Comment by kurokirasama** [comment]
@fdncred I added the error message when I tried the alias with parenthesis.

**Comment by fdncred** [comment]
i think this will take @jntrnr level intervention. aliases are just weird and i don't understand them well enough to fix code if it's broken there. :(

**Comment by pat** [comment]
Perhaps the source of this is different, but I'm seeing the same problem with _some_ (but not all) of the git aliases I have set up.

```
# this works:
alias gpom = git pull origin main
# these do not:
alias gco = git checkout
alias gcom = git checkout main
```

```
$> gcom

Error: nu::shell::external_command (link)

  × External command
   ╭─[entry #20:1:1]
 1 │ gcom
   · ──┬─
   ·   ╰── can't run executable
   ╰────
  help: No such file or directory (os error 2)
```

**Comment by gipsyh** [comment]
> Perhaps the source of this is different, but I'm seeing the same problem with _some_ (but not all) of the git aliases I have set up.
> 
> ```
> # this works:
> alias gpom = git pull origin main
> # these do not:
> alias gco = git checkout
> alias gcom = git checkout main
> ```
> 
> ```
> $> gcom
> 
> Error: nu::shell::external_command (link)
> 
>   × External command
>    ╭─[entry #20:1:1]
>  1 │ gcom
>    · ──┬─
>    ·   ╰── can't run executable
>    ╰────
>   help: No such file or directory (os error 2)
> ```
> 
> In case it's relevant, I'm using version 0.61.0.

It seems worked in latest([b4f8798](https://github.com/nushell/nushell/commit/b4f8798a3adb071d761dbeac9efa3e885c6eeeb2))

**Comment by kurokirasama** [comment]
found a workaround:
```
def gg [...search: string] {
  ddgr -n 5 ($search | str collect ' ')
}
```

**Comment by filaretov** [comment]
Just as a note, these examples do seem to work in 0.61.1.

**Comment by SirZenith** [comment]
This problem seems solved after I upgrade nushell from 0.61.0 to 0.65.0.

**Comment by fdncred** [comment]
closing. please re-open if it's not resolved.
